### PR TITLE
Testing: actually run the playbook & install policycoreutils (for restorecon)

### PR DESCRIPTION
**Pull request by martbhell**
install package so that the restorecon command is available

Also copy in memtest.0 to the DocumentRoot of the web server (so it can be fetched during PXE boot)

### CODE REVIEW

**Comment by oscarkraemer** [comment]
Alright now I can't follow the red thread. I naively though that only one PR would be needed for the memtest stuff. Now I think that a jira task is needed so that it can be followed up and can be reviewed in the end of the sprint. It would probably be a good idea to link the already resolved PR in the jira task.

**Comment by martbhell** [comment]
no problemo: #CCCP-2325

File: tasks/main.yml [hunk 1]
```diff
@@ -37,6 +37,11 @@
   when: groups[dhcp_pxe_group] is defined
   notify:
     - restart dhcpd
+    
+- name: install restorecon for selinux
```
Comment (oscarkraemer) [inline tasks/main.yml]: Why is this needed?
